src/css/blocks/stars.css

```diff
@@ -0,0 +1,35 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('../../images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 > :nth-child(1) {
+  background-image: url('../../images/star-active.svg');
+}
+
+.stars--2 > :nth-child(-n + 2) {
+  background-image: url('../../images/star-active.svg');
+}
+
+.stars--3 > :nth-child(-n + 3) {
+  background-image: url('../../images/star-active.svg');
+}
+
+.stars--4 > :nth-child(-n + 4) {
+  background-image: url('../../images/star-active.svg');
+}
+
+.stars--5 > :nth-child(n + 0) {
+  background-image: url('../../images/star-active.svg');
+}
```

Comment:
You can merge all this selectors and set the property, that is actually the same for all of them, in one place:
```suggestion
.stars--1 > :nth-child(1),
.stars--2 > :nth-child(-n + 2),
.stars--3 > :nth-child(-n + 3),
.stars--4 > :nth-child(-n + 4),
.stars--5 > :nth-child(n + 0) {
  background-image: url('../../images/star-active.svg');
}
```